tests/confreq-rs-xml-nval_comment/OPS/package.opf

```diff
@@ -4,14 +4,14 @@
     <dc:coverage>Publication Resources</dc:coverage>
     <dc:creator>Dave Cramer</dc:creator>
     <dc:date>2021-01-05</dc:date>
-    <dc:description>The spine XML is invalid. The reading system must not display the navigation document.</dc:description>
-    <dc:identifier id="pub-id">confreq-rs-xml-nval_invalid</dc:identifier>
+    <dc:description>This is a test of XML processing. The nav doc should not appear in the spine because the itemref is commented out.</dc:description>
+    <dc:identifier id="pub-id">confreq-rs-xml-nval_comment</dc:identifier>
     <dc:language>en</dc:language>
-    <dc:title>Invalid XML navigation document</dc:title>
+    <dc:title>XML Processing of Comments</dc:title>
     <link rel="dcterms:rights" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document"/>
     <link rel="dcterms:rightsHolder" href="https://www.w3.org"/>
     <meta property="dcterms:isReferencedBy">https://www.w3.org/TR/epub-rs-33/#confreq-rs-xml-nval</meta>
-    <meta property="dcterms:modified">2021-01-05T00:00:00Z</meta>
+    <meta property="dcterms:modified">2021-11-04T00:00:00Z</meta>
```

Comment:
We decided not to increment this; it should match line 6. (Feel free to change them both if you want.)

Reply:
Actually, since then, we decided to update the modified field regularly... so this comment should be moot.